src/Nova/Fields/FeatureCollectionMap.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Wm\WmPackage\Nova\Fields;
+
+use Laravel\Nova\Fields\Text;
+
+class FeatureCollectionMap extends Text
+{
+    protected $modelType;
+
+    /**
+     * Create a new field.
+     *
+     * @param  string  $name
+     * @param  string|callable|null  $attribute
+     * @return void
+     */
+    public function __construct($name, $attribute = null, ?callable $resolveCallback = null)
+    {
+        parent::__construct($name, $attribute, $resolveCallback);
+
+        // Configura automaticamente il field
+        $this->configureField();
+    }
+
+    /**
+     * Set the model type for hiking routes.
+     *
+     * @return $this
+     */
+    public function forHikingRoute()
```

Comment:
non va bene questo componente renderizza geojson puo essere utilizzato in contesti dove non esiste ne hikingroute ne poles come ti ho detto in call il modello lo puoi desumere dalla risorsa